apps/pom.xml

```diff
@@ -516,9 +516,9 @@
 					<version>2.6</version>
 				</plugin>
 				<plugin>
-					<groupId>com.coveo</groupId>
+					<groupId>com.spotify.fmt</groupId>
```

Comment:
At a quick glance there's also https://github.com/diffplug/spotless/tree/main/plugin-maven#google-java-format which has a bit more configurable options, but not sure if we want/need anything it offers.

Reply:
Hmm. I saw that `fmt-maven-plugin` _includes_ `google-java-format` and assumed that the latter was responsible for the formatting, but I haven't looked too closely. Maybe it's a negligible difference in the end?

Reply:
Mistakenly linked to a subsection in the readme; this plugin is called Spotless and also contains google-java-plugin amongst others (and more configuration options)